Rubberduck.VBEEditor/Selection.cs

```diff
@@ -65,7 +65,7 @@ public bool Contains(Selection selection)
         public bool IsSingleCharacter => StartLine == EndLine && StartColumn == EndColumn;
 
         public Selection PreviousLine => StartLine == 1 ? Home : new Selection(StartLine - 1, 1);
-        public Selection NextLine => StartLine == 1 ? Home : new Selection(StartLine + 1, 1);
```

Comment:
are you sure this doesn't break stuff somewhere else?

Reply:
Yeah. Prev/Next were added for AC when I noticed I always wanted the previous and/or next line... They should prolly be both removed.